obfuscate/atm.js

```diff
@@ -1 +1,37 @@
\ No newline at end of file
+var checkingDepositButton = $("#checking_deposit")
+var checkingWithdrawlButton = $("#checking_withdraw")
+var checkingUserInput = $("#checking_amount")
+var checkingTotal = 0
+var checkingDisplay = $("#checking_balance")
+
+var savingDepositButton = $("#savings_deposit")
+var savingWithdrawlButton = $("#savings_withdraw")
+var savingUserInput = $("#savings_amount")
+var savingTotal = 0
+var savingDisplay = $("#savings_balance")
```

Comment:
defining all your vars up top makes the code super easy to read. If you wanted to get fancy, you could combine these variables into an object, e.g:

```js
var accounts = {
  checking: {
    depositButton: ,
    balance: 0
  } 
  savings: ... etc
}
```